sonic-net/sonic-buildimage
# [AS9726-32D] support show system-health

#### Why I did it
support `show system-health` on AS9726-32D model.

#### How I did it
Give a SystemHealth config json, ensure all required systemLED APIs are implemented, Fan, PSU info can be fetched correctly.

#### How to verify it
Make sure all Hardware checks are passed. 
Note : PSU1 wasn't powered on DUT.
```
root@as9726-32d-2:/home/admin# show system-health detail
System status summary

  System status LED  ControlledByFPGA
  Services:
    Status: OK
  Hardware:
    Status: OK

System services and devices monitor list

Name                        Status    Type
--------------------------  --------  ----------
as9726-32d-2                OK        System
rsyslog                     OK        Process
root-overlay                OK        Filesystem
var-log                     OK        Filesystem
routeCheck                  OK        Program
diskCheck                   OK        Program
container_memory_telemetry  OK        Program
FAN-1F                      OK        Fan
FAN-1R                      OK        Fan
FAN-2F                      OK        Fan
FAN-2R                      OK        Fan
FAN-3F                      OK        Fan
FAN-3R                      OK        Fan
FAN-4F                      OK        Fan
FAN-4R                      OK        Fan
FAN-5F                      OK        Fan
FAN-5R                      OK        Fan
FAN-6F                      OK        Fan
FAN-6R                      OK        Fan
PSU-1 FAN-1                 OK        Fan
PSU-2 FAN-1                 OK        Fan
PSU 2                       OK        PSU

System services and devices ignore list

Name               Status    Type
-----------------  --------  -------
container_checker  Ignored   Service
psu.temperature    Ignored   Device
PSU 1              Ignored   Device
asic               Ignored   Device
```

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog

[Accton] support show system-health on AS9726-32D

#### A picture of a cute animal (not mandatory but encouraged)
![Alt Text](https://media.giphy.com/media/viptn1Iwktgkg/giphy.gif)
